env/iasConfig_test.go

```diff
@@ -72,6 +115,20 @@ func setTestEnv(vcapServices string) error {
 	return nil
 }
 
+func setK8sTestEnv(secretPath string) error {
+	err := os.Setenv("KUBERNETES_SERVICE_HOST", "0.0.0.0")
+	if err != nil {
+		return fmt.Errorf("error preparing test: could not set env KUBERNETES_SERVICE_HOST: %w", err)
+	}
```

Comment:
If you are on go 1.17 you can use https://pkg.go.dev/testing#T.Setenv to temporarily set an environment variable to a certain value.

Reply:
good one!
But we still have to support go 1.16